minimesos/src/main/java/com/containersol/minimesos/cluster/MesosCluster.java

```diff
@@ -156,6 +164,8 @@ public void state(PrintStream out, String agentContainerId) {
 
     /**
      * Destroys the Mesos cluster and its containers
+     *
+     * @param factory TODO(mwl)
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://ci.infra.container-solutions.com/sonar/coding_rules#rule_key=squid%3AS1135)